runtime/ast/position.go

```diff
@@ -40,12 +40,13 @@ func (position Position) Shifted(length int) Position {
 	}
 }
 
-func (position Position) Hash() (result uint32) {
```

Comment:
Was this not used?

Reply:
It had been previously used / was previously needed when we used a persistent map/set library that used this function for the key. We replaced it and removed the dependency a while ago, and this was a left-over